config.json.example

```diff
@@ -1,15 +0,0 @@
-{
-  "token": "",
-  "join_message": "I has joined to do stuff 🌟",
-  "owners": [
-    86477779717066752
-  ],
-  "prefix": [
-    "!"
-  ],
-  "activity": "games!",
-  "activity_type": "playing|watching|listening|competing",
-  "status_type": "online|idle|dnd",
-  "alexflipnote_api": "nice_token_you_got_here",
-  "version": "2.0.2"
-}
```

Comment:
Why is this file deleted? This is an important file as it shows the developer how to create their own config.json. Please put it back.

Reply:
If you renamed the file to create your local config.json, you should have made a copy instead.

Reply:
I added the config file back into the repository and pushed the changes.